relay-spans/src/v2_to_v1.rs

```diff
@@ -1055,8 +1055,8 @@ mod tests {
           "trace_id": "89143b0763095bd9c9955e8175d1fb23",
           "status": "unknown",
           "data": {
-            "gen_ai.agent.name": "Seer",
-            "gen_ai.system": "openai"
+            "gen_ai.system": "openai",
+            "gen_ai.agent.name": "Seer"
```

Comment:
Can you revert this? They were alphabetical before.

Reply:
as discussed in slack, i will keep this change.